lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1213,6 +1198,7 @@ def __init__(
         # create tenant dir so that the /loki/api/v1/rules endpoint returns "no rule groups found"
         # instead of "unable to read rule dir /loki/rules/fake: no such file or directory"
         if self.container.can_connect():
+            logger.debug("Cannot connect to container to make alert rules path!")
```

Comment:
Why do we even need to log this if we assume that we will create the file later?